### ISSUE TITLE
Import - Skipped file since its format is not supported (edn file)

### ISSUE THREAD

**Issue by aforank** [logseq/db-test#196]
### Search first

- [X] I searched and no similar issues were found

### What Happened?

I imported my existing graph for the first time, and one of the most noticeable issues I encountered is that the Edn files were not imported. It seems these files contain my highlights from a PDF. However, when I open the corresponding PDF files, my highlights no longer appear, and any references to these highlights are also broken (appears in yellow), as shown below.

![image](https://github.com/user-attachments/assets/ed6bd659-493d-41cc-9efb-9bfdb5b351f5)

![image](https://github.com/user-attachments/assets/b36569a7-b278-4039-a0d7-0b707f020daf)

EDN file content

`{:highlights [{:id #uuid "647f43c6-7439-4c97-851f-a4286c04b11e",
               :page 73,
               :position {:bounding {:x1 11,
                                     :y1 4,
                                     :x2 674,
                                     :y2 359,
                                     :width 682.0000000000001,
                                     :height 383.62500000000006},
                          :rects (),
                          :page 73},
               :content {:text "[:span]", :image 1686062020889},
               :properties {:color "yellow"}}],
 :extra {:page 81}}`

100's of similar errors were shown after import.

### Reproduce the Bug

Import a page with any PDF highlighting.

### Are you willing to submit a PR? If you know how to fix the bug.

- [ ] I'm willing to submit a PR (Thank you!)

**Comment by logseq-cldwalker** [comment]
Hi. This is a known todo as described in https://github.com/logseq/docs/blob/feat/db/db-version.md#importer-todos. We'll get to this at some point. The more people that upvote or comment this issue, the sooner we'll address it

**Comment by logseq-cldwalker** [comment]
In the meantime I have improved the import ux so that only notification is displayed for all ignored files

**Comment by yingzhu146** [comment]
Basically the main missing thing for me to be able to use the DB version. So many PDF annotations :)

**Comment by Sininho** [comment]
> Basically the main missing thing for me to be able to use the DB version. So many PDF annotations :)

Yes, it is a big problem for me too ;-( Large parts of my graph are unusable because they rely on pdf annotations...

**Comment by kerim** [comment]
I just tested this, creating the following graph with one PDF attachment and two annotations, and then trying to import it. The two annotations are marked as invalid blocks on import.

[annotation test.zip](https://github.com/user-attachments/files/21172106/annotation.test.zip)

**Comment by logseq-cldwalker** [comment]
This should be addressed and available on test.logseq.com in 10 min. If you're having an issue with specific pdf highlights, please open a new issue and share the relevant pdf, edn and md files. If those files are sensitive then please email them to me at gabriel@logseq.com

@kerim Thanks for sharing your test graph as it helped me catch a bug with file names having special characters
